#!/usr/bin/env python
PACKAGE = "autonomus"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("follow_distance",   double_t, 0, "Desired follow distance (cm)",       50.0,  0.0, 500.0)
gen.add("safe_distance",     double_t, 0, "Safe distance for obstacle avoidance",30.0,  0.0, 500.0)
gen.add("max_linear_speed",  double_t, 0, "Max linear speed",                   0.5,   0.0, 2.0)
gen.add("max_angular_speed", double_t, 0, "Max angular speed",                  1.0,   0.0, 5.0)

exit(gen.generate(PACKAGE, "autonomus", "AutonomousConfig"))
